{"name":"Create: Failure","fullName":"TestUserPgRepoSuiteRunner/UserPgRepoSuit/TestUserPgRepo_CreateFailure","status":"passed","statusDetails":{"message":"","trace":""},"start":1728207674850,"stop":1728207674850,"uuid":"2114a1e2-83c7-11ef-8b52-be2bab8c44a2","historyId":"617a2808db6508c1237a522a82415da6","testCaseId":"cd57d8d6edc53d1c69c8ac27ff125bd0","labels":[{"name":"language","value":"go1.23.0"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"MBP-Pavel-2.netis.cc"},{"name":"thread","value":"TestUserPgRepoSuiteRunner/UserPgRepoSuit/TestUserPgRepo_CreateFailure"},{"name":"suite","value":"UserPgRepoSuit"},{"name":"package","value":"cookdroogers/internal/repo/pg"},{"name":"tag","value":"UserPgRepo"}],"steps":[{"name":"Failure","status":"passed","start":1728207674850,"stop":1728207674850,"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1728207674850,"stop":1728207674850,"parameters":[{"name":"Error","value":"sql: connection is already closed: error in postgres repo"},{"name":"Target","value":"error in postgres repo"}]}]}]}